### PR TITLE
[a11y] Enable tab navigation over SessionList

### PR DESCRIPTION
**Pull request by vinicius73**
* Resolves: #2733
* Target version: master 

### Summary

Enable tab usage on session list popover.

- Add visual feedback about focus
- restore focus on editor after close popover

https://user-images.githubusercontent.com/1561347/182258054-60e7216b-06c4-42b1-ab5e-900fe471e6b9.mp4

https://user-images.githubusercontent.com/1561347/182258071-2f35f429-037b-4500-9a17-4755e7c2ecb6.mp4

### CODE REVIEW

File: src/components/Editor/SessionList.vue
Comment (max-nextcloud) [inline src/components/Editor/SessionList.vue:146]: since there is only a single command in your chain i think you could do
```suggestion
			this.$editor.commands.focus()
```

**Review by max-nextcloud — APPROVED** [review]
Codewise looks good to me... just one minor comment. I still approve so you can fix if you want and then go ahead without another review.

I did not try this myself - your screen capture seems to make sense.

File: src/components/Editor/SessionList.vue
Comment (max-nextcloud) [reply src/components/Editor/SessionList.vue:146]: https://tiptap.dev/api/commands#execute-a-command

File: src/components/Editor/SessionList.vue
Comment (juliushaertl) [inline src/components/Editor/SessionList.vue:54]: I think we should rather put the initial focus on https://github.com/nextcloud/text/pull/2771/files#diff-9e0a2a342f91d6277778b1bb6b17da4f37cdab38ef53c98a7c97a84deaf488a5R40 so that the content gets read and maybe give this a descriptive label then

File: src/components/Editor/SessionList.vue
Comment (juliushaertl) [inline src/components/Editor/SessionList.vue:27]: Maybe we can additionally catch tab/escape and prevent the default event handling and instead just revert the focus.

File: src/components/Editor/SessionList.vue
Comment (juliushaertl) [inline src/components/Editor/SessionList.vue:146]: Should this rather go back to focus on the button after closing?

**Comment by juliushaertl** [comment]
@vinicius73 Any thoughts about using something like in https://github.com/nextcloud/nextcloud-vue/pull/2654 ?

Could also be fixed upstream globally then https://github.com/nextcloud/nextcloud-vue/issues/2860

File: src/components/Editor/SessionList.vue
Comment (vinicius73) [reply src/components/Editor/SessionList.vue:146]: Thanks!

Comment (vinicius73) [reply src/components/Editor/SessionList.vue]: I don't know. But I am open for ideas.

**Comment by vinicius73** [comment]
> @vinicius73 Any thoughts about using something like in [nextcloud/nextcloud-vue#2654](https://github.com/nextcloud/nextcloud-vue/pull/2654) ?

Looks interesting, I will play around this a bit.

**Comment by vinicius73** [comment]
Works like a charm @juliushaertl 
I will do some upstream of this, maybe we can drop this PR.

**Comment by vinicius73** [comment]
Waiting for next release of `@nextcloud/vue`

https://github.com/nextcloud/nextcloud-vue/pull/2941

**Pull request closed without merging**
